projects/gnome.org/libxslt/package.yml

```diff
@@ -0,0 +1,36 @@
+distributable:
+  url: https://download.gnome.org/sources/libxslt/{{ version.marketing }}/libxslt-{{ version }}.tar.xz
+  strip-components: 1
+
+versions:
+  github: GNOME/libxslt/tags
+
+dependencies:
+  gnome.org/libxml2: '*'
+
+build:
+  dependencies:
+    freedesktop.org/pkg-config: ^0.29
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+  script: |
+    ./configure $ARGS
+    make --jobs {{ hw.concurrency }}
+    make install
+  env:
+    ARGS:
+      - --prefix={{prefix}}
+      - --without-python  # mysterious build failure in GHA macOS-11
```

Comment:
the comment is copy pasted from libxml2. Does it apply? Or did you just YOLO paste?

Reply:
Applies here as well. Without it it uses Python and crashes the build

Reply:
But yes I checked the `libxml2` build first because the docs say the build instructions are very similar.

Reply:
k good stuff ☺️